.github/workflows/build-ci.yml

```diff
@@ -39,40 +42,48 @@ jobs:
                     sleep 1
                     done
                     sudo docker exec --tty mongodb $MONGOSH_BIN 127.0.0.1:27017 --eval "rs.initiate({\"_id\":\"rs\",\"members\":[{\"_id\":0,\"host\":\"127.0.0.1:27017\" }]})"
+
             -   name: Show MongoDB server status
                 run: |
                     if [ "${{ matrix.mongodb }}" = "4.4" ]; then MONGOSH_BIN="mongo"; else MONGOSH_BIN="mongosh"; fi
                     docker exec --tty mongodb $MONGOSH_BIN 127.0.0.1:27017 --eval "db.runCommand({ serverStatus: 1 })"
-            -   name: "Installing php"
+
+            -   name: Installing php
                 uses: shivammathur/setup-php@v2
                 with:
                     php-version: ${{ matrix.php }}
                     extensions: curl,mbstring,xdebug
                     coverage: xdebug
                     tools: composer
+
             -   name: Show PHP version
                 run: php -v && composer -V
+
             -   name: Show Docker version
                 run: if [[ "$DEBUG" == "true" ]]; then docker version && env; fi
                 env:
                     DEBUG: ${{secrets.DEBUG}}
+
             -   name: Download Composer cache dependencies from cache
                 id: composer-cache
                 run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
             -   name: Cache Composer dependencies
                 uses: actions/cache@v3
                 with:
                     path: ${{ steps.composer-cache.outputs.dir }}
                     key: ${{ matrix.os }}-composer-${{ hashFiles('**/composer.json') }}
                     restore-keys: ${{ matrix.os }}-composer-
+
             -   name: Install dependencies
                 run: |
                     composer update --no-interaction $([[ "${{ matrix.mode }}" == low-deps ]] && echo ' --prefer-lowest --prefer-stable')
+
             -   name: Run tests
-                run: |
-                    ./vendor/bin/phpunit --coverage-clover coverage.xml
+                run: composer test:coverage
```

Comment:
Use a composer script in `build-ci` workflow.

Reply:
It makes things a little more complicated if we want to add new options to this command.

Reply:
reverted.